nextflow.config

```
@@ -268,6 +268,7 @@ manifest {
     nextflowVersion = '!>=24.04.2'
     version = '1.3.0dev'
     doi = '10.5281/zenodo.10456900,10.5281/zenodo.11267672'
+    manifest.icon = 'nf-core/fastquorum/icon.png'
```

```suggestion
    icon = 'nf-core/fastquorum/icon.png'
```

Just `icon` because it's already in the `manifest` block of the config,